src/Symfony/Component/DependencyInjection/Dumper/GraphvizDumper.php

```diff
@@ -173,7 +173,7 @@ private function findNodes()
         foreach ($container->getServiceIds() as $id) {
             $service = $container->get($id);
 
-            if (in_array($id, array_keys($container->getAliases()))) {
+            if (array_key_exists($id, $container->getAliases())) {
```

Comment:
this does not have exactly the same behaviour (as `in_array` was not executed in strict mode). However, I don't think it'll break anything